cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/UnmanagedClusterWizard.tsx

```diff
@@ -55,7 +51,7 @@ function UnmanagedClusterWizard() {
             <div cds-layout="vertical align:stretch">
                 <div className="section-raised" cds-layout="horizontal align:vertical-center p:md">
                     <div>Simple configuration</div>
-                    <CdsButton action="outline" cds-layout="align:right" size="sm" onclick={changeSettings}>
+                    <CdsButton action="outline" cds-layout="align:right" size="sm" onClick={() => setUseBasicSettings(!useBasicSettings)}>
```

Comment:
I'm curious about this change. I kinda like `changeSettings` (or even better something like `toggleBasicAdvanced`), rather than a closure here.

Reply:
+1

Reply:
If I am understanding it right.  onClick takes in a function as expected. I don't see anything wrong with this.  I agree with the naming toggleBasicAdvanced is better.